couchbase/operations/management/query_index_drop.cxx

```diff
@@ -75,7 +75,7 @@ query_index_drop_request::make_response(error_context::http&& ctx, const encoded
                         bucket_not_found = true;
                         break;
                     case 12004: /* IKey: "datastore.couchbase.primary_idx_not_found" */
-                    case 12006: /* IKey: "datastore.couchbase.keyspace_count_error" */
+                    case 12016: /* IKey: "datastore.couchbase.keyspace_count_error" */
```

Comment:
12006 is key_space_count_error:

https://github.com/couchbase/query/blob/edb8b6ebd70d2a1d886cecaab4e4f2ba64d92965/errors/codes.go#L299

https://github.com/couchbase/query/blob/edb8b6ebd70d2a1d886cecaab4e4f2ba64d92965/errors/ds_couchbase.go#L45

Reply:
I mean, could you fix the comment too?

Change it to `"datastore.couchbase.index_not_found"`

https://github.com/couchbase/query/blob/edb8b6ebd70d2a1d886cecaab4e4f2ba64d92965/errors/ds_couchbase.go#L96